todo_app/templates/index.html

```diff
@@ -2,17 +2,29 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
+<form action="/item" method="post" class="todo-form" id="todoForm">
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="todo-header" id="todoHeader">To-Do App</h1>
+    <p class="todo-header-para" id="todoHeaderPara">Things to-do app.</p>
   </div>
-
+  <!-- <div class="jumbotron"> -->
+    <h2 class="add-item-input add-item-submit" id="addItemInputSubmit">Add Item</h1>
+    <input type="text" name="item-input" value=""></input>
```

Comment:
```suggestion
    <input type="text" name="item-input" value="" required></input>
```

just like so

Reply:
Thanks very much - have added in commit a943886